src/script_lua.c

```diff
@@ -1549,7 +1562,7 @@ static void luaCreateArray(lua_State *lua, robj **elev, int elec) {
 
 /* The following implementation is the one shipped with Lua itself but with
  * rand() replaced by serverLrand48(). */
-static int redis_math_random (lua_State *L) {
+static int server_math_random (lua_State *L) {
   /* the `%' avoids the (rare) case of r==1, and is needed also because on
      some systems (SunOS!) `rand()' may return a value larger than RAND_MAX */
   lua_Number r = (lua_Number)(serverLrand48()%REDIS_LRAND48_MAX) /
```

Comment:
Rename REDIS_LRAND48_MAX?

```suggestion
  lua_Number r = (lua_Number)(serverLrand48()%VALKEY_LRAND48_MAX) /
```

Reply:
good catch. done.